src/views/User.vue

```diff
@@ -26,18 +35,79 @@ import {store} from "../store";
 })
 export default class User extends Vue {
 	@Prop() userName!: string;
+	@Prop({
+		required: false,
+		default: null,
+		type: Number,
+	}) after!: number | null;
```

Comment:
📝 自信なしポイント1

「次のページ」に行くために起点となるカーソル情報が必要。
カーソル情報は記事IDにしたいところだが、createdでオーダーしているのでcreatedにしている。

・・が、多分これはDocumentReferenceに差し替えられそう。
https://firebase.google.com/docs/firestore/query-data/query-cursors?hl=ja

Reply:
Referenceだとダメだ。getしちゃえばいけるんだが。
createdでやる方がいいのか。そうなのか・・？

Reply:
URLとしてださいのでそれはやりたくないなぁ。getしちゃう

Reply:
またクエリが増えたがこうなるようには書けた

- 最初: http://localhost:8080/tsugehara で1ページ目
- 2ページ目: http://localhost:8080/tsugehara?since=2ZHHVmgLX0AOvhDUVeSB&page=1
- 3ページ目: http://localhost:8080/tsugehara?since=J0vLgA1kpUJILHleXlLN&page=2
- 3ページ目から2ページ目に戻る: http://localhost:8080/tsugehara?since=hTcij6vdWhklpYyiuAvy&order=asc&page=1

Reply:
pageをいらないようにするべきなのかが判断つかんので寝かす
https://github.com/shinonomekazan/blog/pull/4